app/styles/variables.css

```diff
@@ -1,7 +1,7 @@
 :root {
   --color-white: #fff;
   --color-offwhite: #fffbfb;
-  --color-greywhite: #d0e7ff;
+  --color-greywhite: #d7d6d6;
```

Comment:
will it not affect the colors in other places wherever `#d0e7ff`  was looking good